onvm/onvm_mgr/onvm_init.h

```diff
@@ -103,6 +103,8 @@
 #define NO_FLAGS 0
 
 #define ONVM_NUM_RX_THREADS 1
+/* Number of non RX/TX threads in manager */
+#define ONVM_NUM_MGR_THREADS 1
```

Comment:
ONVM_NUM_STATS_THREADS would be better as we don't have other mgr specific threads

Reply:
@koolzz - the issue with that is I doubt we will ever add more STATS threads, but we might add other types of threads to the manager (e.g., an SDN communication thread) which are not related to statistics. 

I agree `ONVM_NUM_RX_THREADS` is a little confusing because I think of both RX and TX threads as being part of the manager

Any other name ideas?  Maybe `ONVM_NUM_MGR_AUX_THREADS` to refer to "auxiliary" functions in the manager (stats, etc)?